docs/releasenotes/archive/service/year2021.md

```diff
@@ -6,3 +6,770 @@ description: This is an archive of 2021 Sumo Logic service release notes.
 ---
 
 This is a 2021 archive of Service Release Notes. The current Service Release Notes are [here](/docs/releasenotes/service).
+
+---
+## December 17, 2021 (Search)
+
+New - Keyboard shortcuts are now available to generate [comments in Search queries](https://help.sumologic.com/05Search/Get-Started-with-Search/Search-Basics/Comments-in-Search-Queries "Comments in Search Queries"). Use **command + /** on macOS or **control + /** on Windows.
+
+---
+## December 9, 2021 (Collection)
+
+New - Our [Cloud-to-Cloud Integration Framework](https://help.sumologic.com/03Send-Data/Sources/02Sources-for-Hosted-Collectors/Cloud-to-Cloud_Integration_Framework) has a new Source. The [Google Workspace Source](https://help.sumologic.com/03Send-Data/Sources/02Sources-for-Hosted-Collectors/Cloud-to-Cloud_Integration_Framework/Google_Workspace_Source "Google Workspace Source") collects data from the Google Workspace [Users API](https://developers.google.com/admin-sdk/directory/reference/rest/v1/users/list).
+
+Update - The [Okta Source](https://help.sumologic.com/03Send-Data/Sources/02Sources-for-Hosted-Collectors/Cloud-to-Cloud_Integration_Framework/Okta_Source "Okta Source"), in addition to [System Log](https://developer.okta.com/docs/reference/api/system-log/) data, now supports collecting [User Inventory](https://developer.okta.com/docs/reference/api/users/ "https://developer.okta.com/docs/reference/api/users/") data.
+
+---
+## November 30, 2021 (Apps)
+
+New - As an integration partner with Amazon, we are pleased to announce our new [Amazon Inspector App](/docs/integrations/amazon-aws/inspector). Amazon Inspector is an automated vulnerability management service that continually scans Amazon EC2 and container images for software vulnerabilities and network exposures. The Sumo Logic Inspector App helps reveal trends and identify anomalies from these findings.
+
+![](https://help.sumologic.com/)![inspector-app-overview.png](https://help.sumologic.com/@api/deki/files/12359/inspector-app-overview2.png?revision=1)
```

Comment:
This produced an empty image: ![](https://help.sumologic.com/). 

The second part is ok though ![inspector-app-overview.png](https://help.sumologic.com/@api/deki/files/12359/inspector-app-overview2.png?revision=1)